ScreensaverHero.hx

```diff
@@ -0,0 +1,57 @@
+package;
+
+    import flixel.FlxSprite;
+    import flixel.util.FlxColor;
+    import flixel.FlxG;
+
+    class ScreensaverHero extends FlxSprite {
+        
+        public function new(x:Int = 0, y:Int = 0){
+            super(x,y);
+            super.makeGraphic(30, 30);
+        
+        }
+
+
+        override public function update(elapsed:Float){
+            
+            if (FlxG.keys.justPressed.SPACE){
+                this.velocity.x *= -1;
+                this.velocity.y *= -1;
+                }
+            
+            if (this.x <= (FlxG.width/2) && this.y <= (FlxG.height/2)){
+                this.color = 0xFF1500FF;
+            }
+            else if (this.x > (FlxG.width/2) && this.y <= (FlxG.height/2)){
+                this.color = 0xFFFFFB00;
+            }
+            else if (this.x <= (FlxG.width/2) && this.y > (FlxG.height/2)){
+                this.color = 0xFF16B900;
+            }
+            else if (this.x > (FlxG.width/2) && this.y > (FlxG.height/2)){
+                this.color = 0xFFFF0000;
```

Comment:
These colors are also great examples of "magic numbers" that we'd rather store inside variables.

Let's fix em up before we call the story done.